src/main/java/com/minecolonies/client/gui/WindowBuildTool.java

```diff
@@ -115,13 +115,16 @@
      */
     private static final String HUT_PREFIX = ":blockHut";
 
-    private static final BlockPos DEFAULT_POS = new BlockPos(0, 0, 0);
-
     private static final int POSSIBLE_ROTATIONS = 4;
     private static final int ROTATE_RIGHT       = 1;
     private static final int ROTATE_LEFT        = 3;
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)